schema.sql

```diff
@@ -8,4 +8,5 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL,
-);
\ No newline at end of file
+    species VARCHAR
```

Comment:
- [OPTIONAL] I think you should add the column by entering this command `   ALTER TABLE animals ADD species varchar(255)`  not by manually editing the file, please kindly fix that,